Domain/Models/Bookings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Models
+{
+    public  class Bookings:BaseEntity
```

Comment:
class name should be Singular (Booking)

and ideally formatting: public class Booking : BaseEntity